custom_components/browser_mod/mod_view.py

```diff
@@ -66,6 +68,19 @@ async def async_setup_view(hass: HomeAssistant):
         for r in resources.async_items():
             if r["url"].startswith(FRONTEND_SCRIPT_URL):
                 frontend_added = True
+                if not r["url"].endswith(version):
```

Comment:
Does this actually ever happen?

Reply:
Yes, on upgrade. So this check is to make sure that the cache buster version string is teh latest version for resource added by a previous version.